src/spec/integration/subscriptions.spec.coffee

```diff
@@ -0,0 +1,71 @@
+{SphereClient} = require '../../lib/main'
+Config = require('../../config').config
+testConfig = require('../../test-config').config
+Promise = require 'bluebird'
+debug = require('debug')('spec-integration:products')
+
+newSubscription =
+  key: 'testKey'
+  destination:
+    type: 'IronMQ'
+    uri: "https://#{testConfig.iron_mq_url}/3/projects/#{testConfig.iron_project_id}/queues/test-queue/webhook?oauth=#{testConfig.iron_token}"
```

Comment:
_https_ should not be hard coded. and it's much more better that the user provides the full uri. There is no assurance that the format you using to construct the uri will remain constant.